Zend/zend_API.h

```diff
@@ -1881,6 +1896,36 @@ static zend_always_inline int zend_parse_arg_str_or_array_ht(
 	return 1;
 }
 
+static zend_always_inline int zend_parse_arg_str_or_long(zval *arg, zend_string **dest_str, zend_long *dest_long,
+	zend_bool *is_null, int allow_null)
+{
+	if (allow_null) {
+		*is_null = 0;
+	}
+	if (EXPECTED(Z_TYPE_P(arg) == IS_STRING)) {
+		*dest_str = Z_STR_P(arg);
+		*dest_long = 0;
```

Comment:
This assignment is not needed (dest_long == 0 cannot be used to distinguish whether the param is long or not).